src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabServerConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import jakarta.inject.Inject;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+public class GitLabServerConfig extends PageAreaImpl {
+
+    @Inject
+    public GitLabServerConfig(Jenkins jenkins) {
+        super(jenkins, "/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers");
+    }
+
+    public void configureServer(String url) {
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/name")).sendKeys("servername");
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).clear();
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).sendKeys(url);
```

Comment:
you shouldn't need to be using the full form here - things should be relative to the context (which is passed in the contructor) so  
`find(by.path("servers/name")).sendKeys("servername");`

but also - these should likely be controlles.  - is this a repeatable option - can it occur more than once?

Reply:
I put the whole path because it doesn’t find the element if I use `find(by.path("servers/name")).sendKeys("servername");`
I just tried again and it’s the same error I had before
`Method threw 'org.openqa.selenium.NoSuchElementException' exception.`

Reply:
because you are using find and not a control (the later will be relative to this PageArea)

Reply:
It still doesn't work if I don't put the whole path, it doesn't find the element. I modified to use `Control` but with the full path...